python/GafferUITest/ExamplesTest.py

```diff
@@ -185,5 +185,25 @@ def testExampleFilesExist( self ) :
 
 		self.assertExampleFilesExist()
 
+	def testExamplesLoadedAsNew( self ) :
+
+		examples = GafferUI.Examples.registeredExamples()
+		self.assertTrue( len(examples) > 0 )
+
+		_, e = examples.items()[-1]
+
+		a = Gaffer.ApplicationRoot()
+
+		def load() :
+			GafferUI.Examples.openExample( e["filePath"], a )
+			GafferUI.EventLoop.mainEventLoop().stop()
+
+		GafferUI.EventLoop.addIdleCallback( load )
```

Comment:
@johnhaddon Is this the right way to test this?

Reply:
Can you explain we why you need the idle event stuff here? Honestly though, I'd be happy enough without tests for this - we can't test all the UI code, and I'd rather keep the Examples API in the minimal form it was in before.

Reply:
Sure thing, can remove it - it seemed to need the event loop to be running to add the script, otherwise it moaned that it wasn't running....

Reply:
Gone + pushed